shared/teams/team/nav-header/index.new.tsx

```diff
@@ -206,6 +279,32 @@ export const SubHeader = (props: SubHeaderProps) =>
 const styles = Styles.styleSheetCreate(
   () =>
     ({
+      addInviteAndLinkBox: Styles.platformStyles({
+        common: {
+          flexShrink: 0,
+          padding: Styles.globalMargins.tiny,
+        },
+        isElectron: {
+          borderRadius: 4,
+          boxShadow: `0 2px 10px 0 ${Styles.globalColors.black_20}`,
```

Comment:
`...Styles.desktopStyles.boxShadow`

Reply:
Actually I think you could do this in `desktop/renderer/style.css` to make the hover behavior in the design work.

Reply:
Whoa, I totally missed that that was what was going on with this drop shadow situation